packages/wrangler/src/__tests__/tail.test.ts

```diff
@@ -293,6 +293,51 @@ describe("tail", () => {
       `);
     });
 
+    it("logs console messages and exceptions", async () => {
+      const api = mockWebsocketAPIs();
+      await runWrangler("tail test-worker");
+
+      const event = generateMockRequestEvent();
+      const message = generateMockEventMessage({
+        event,
+        logs: [
+          { message: "some string", level: "log", timestamp: 1234561 },
+          { message: { complex: "object" }, level: "log", timestamp: 1234562 },
+          { message: 1234, level: "error", timestamp: 1234563 },
+        ],
```

Comment:
When this appears in a Worker:
```js
console.log("hello", 123, ["a", "b", "c"], { key: "value" });
console.warn("bye");
```

The following is generated from the trace:
```js
{
  logs: [{
    timestamp: 12345678,
    level: "log",
    message: [
      "hello",
      123,
      ["a", "b", "c"],
      { key: "value" }
    ]
  }, {
    timestamp: 12345679,
    level: "warn",
    message: ["hello"]
  }]
}
```

Note that `message` is always an array. Maybe these tests should also do that as well?